app/models/curiosity_card.rb

```diff
@@ -35,4 +35,12 @@ def publish!
   def unpublish!
     update(published_at: nil)
   end
+
+  def statement_exists?(user)
+    statements.exists?(user_id: user)
```

Comment:
Los queries con foreign keys en active record pueden ser usando las relaciones

```suggestion
    statements.exists?(user: user)
```

ActiveRecord se encarga de la magia